README.md

```diff
@@ -774,11 +854,15 @@ Future.prototype.chain :: Future e a ~> (a -> Future e b) ->        Future e b
 
 </details>
 
-Allows the creation of a new Future based on the resolution value. This is like
-doing `promise.then(x => Promise.resolve(x + 1))`, except that it's lazy, so the
-new Future will not be created until the other one is forked. The function is
-only ever applied to the resolution value; it's ignored when the Future was
-rejected. To learn more about the exact behaviour of `chain`, check out its [spec][FL:chain].
+Sequence a new Future using the resolution value from another. Similarly to
+[`map`](#map), `chain` expects a function to transform the resolution value of
+a Future. But instead of returning the new *value*, chain expects a Future to
+be returned.
+
+The transformation is only applied to the resolution branch: If the Future is
+rejected, the transformation is ignored.
```

Comment:
I suggest replacing “If” with “if” after the colon.

Reply:
:zap: